packager/media/formats/mp4/box_definitions.h

```diff
@@ -328,6 +328,13 @@ struct AC3Specific : Box {
   std::vector<uint8_t> data;
 };
 
+struct MHAConfiguration : Box {
+  DECLARE_BOX_METHODS(MHAConfiguration);
+
+  std::vector<uint8_t> data;
+  std::uint8_t mpeg_h_3da_profile_level_indication;
```

Comment:
Replace std::uint8_t with uint8_t to be consistent

Reply:
Done